backend/bot/MatchReporter.js

```diff
@@ -47,13 +60,21 @@ const createMatchResultMessage = (match) => {
   messageParts.push(`${winningPlayers.join(', ')} just beat ${losingPlayers.join(', ')}.`)
   messageParts.push(`Each winner gets ${winningTeamRatingChange} points, each loser loses ${-losingTeamRatingChange} points.`)
 
-  if (isComedyDuo) {
-    messageParts.push(':marioluigi:')
+  if (suffix) {
+    messageParts.push(suffix)
   }
 
   return messageParts.join(' ')
 }
 
+const getMatchResultPrefixSuffix = (prefixSuffixConfig, winningTeam) => {
```

Comment:
findDecoratorForTeam(decorationConfig, team) ? 🤷‍♂ 

also then, should return a falsy thing...

Reply:
I like the whole `decorator` and `decoration` idea. Gonna go with it.